nixos/modules/services/hardware/fwupd.nix

```
@@ -120,6 +121,14 @@ in {
           Which fwupd package to use.
         '';
       };
+
+      extraDaemonConfigs = mkOption {
```

I think we commonly use singular, so
```suggestion
      extraDaemonConfig = mkOption {
```

?